mlflow/environment_variables.py

```diff
@@ -194,3 +194,7 @@ def get(self):
 MLFLOW_ARTIFACT_UPLOAD_DOWNLOAD_TIMEOUT = _EnvironmentVariable(
     "MLFLOW_ARTIFACT_UPLOAD_DOWNLOAD_TIMEOUT", int, None
 )
+
+MLFLOW_DEFAULT_PREDICTION_DEVICE = _EnvironmentVariable(
+    "MLFLOW_DEFAULT_PREDICTION_DEVICE", str, None
+)
```

Comment:
Following from https://github.com/mlflow/mlflow/pull/7922/files#r1122432221, can we document the use case of the env var and supported frameworks using syntax similar to the other env vars? This gets rendered here: https://mlflow.org/docs/latest/python_api/mlflow.environment_variables.html

Reply:
Whoops - missed that! Addressed